modules/programs/eza.nix

```diff
@@ -4,16 +4,29 @@ with lib;
 
 {
   imports = let
+    msg =
+      "'programs.eza.enableAliases' has been deprecated and replaced with integrations options per shell, ex. 'programs.eza.enableBashIntegration'";
     mkRenamed = opt:
       mkRenamedOptionModule [ "programs" "exa" opt ] [ "programs" "eza" opt ];
-  in map mkRenamed [ "enable" "enableAliases" "extraOptions" "icons" "git" ];
+  in [
+    (map mkRenamed [ "enable" "extraOptions" "icons" "git" ])
+    (mkRemovedOptionModule [ "programs" "eza" "enableAliases" ] msg)
+  ];
 
   meta.maintainers = [ maintainers.cafkafk ];
 
   options.programs.eza = {
     enable = mkEnableOption "eza, a modern replacement for {command}`ls`";
 
-    enableAliases = mkEnableOption "recommended eza aliases (ls, ll…)";
+    enableBashIntegration = mkEnableOption "Bash integration";
```

Comment:
Perhaps these new options should default to true?

I imagine that people typically want these aliases if they enable the eza module and with this change they would have to explicitly enable it for each shell.

Reply:
Not to mention that `enable*Integration` options typically are enabled by default and it would be confusing to have too many exceptions to this rule…